### PR TITLE
Add INT8 quantization doc

### PR DESCRIPTION
**Pull request by hshen14**
8-bit quantization is useful to improve the inference performance. This PR is to add INT8 quantization for Stable Diffusion through Optimum-Intel quantization API on top of Intel Neural Compressor. The sample code is implemented in [Optimum-Intel](https://github.com/huggingface/optimum-intel/tree/main/examples/neural_compressor/text-to-image).

### CODE REVIEW

**Comment by hshen14** [comment]
CC @echarlaix  @michaelbenayoun

**Comment by echarlaix** [comment]
Hi @hshen14, I would prefer for us to wait for the refactorization of both `neural-compressor` and `optimum-intel` before increasing visibility. Also, it would be great to disable the accuracy driven strategy to make things much easier for the quantization of diffusion models, what do you think ?

**Comment by hshen14** [comment]
> Hi @hshen14, I would prefer for us to wait for the refactorization of both `neural-compressor` and `optimum-intel` before increasing visibility. Also, it would be great to disable the accuracy driven strategy to make things much easier for the quantization of diffusion models, what do you think ?

Hi @echarlaix , it might be better to leave users to decide how to use quantization APIs. For simplicity, no accuracy-aware looks better; for accuracy, need accuracy driven strategy. We may provide both options to users and start from simple one.

File: README.md [hunk 1]
```diff
@@ -351,6 +351,11 @@ Textual Inversion is a technique for capturing novel concepts from a small numbe
...
 
 
+## Quantizing Stable Diffusion
+
+Quantization is one of widey-used model compression techniques to accelerate the inference performance. There are two typical approaches: 1) post-training quantization (PTQ), and 2) during-training quantization (DTQ). PTQ is an effective approach to quantizing a model without additional training steps but requires an offline calibration process using a representative dataset to determinate the quantization parameters (e.g., scale and zero point) for the model. DTQ is designed to simulate the quantization error and get recovered as much as possible during training, and quantization-aware training is a well-known approach for DTQ. Optimum-Intel provides the quantization support and sample code on how to enable quantization for Stable Diffusion using [post-training static or dynamic quantization](https://github.com/huggingface/optimum-intel/tree/main/examples/neural_compressor/text-to-image).
```
Comment (patrickvonplaten) [inline README.md:356]: That works with me - should we maybe also add a Python example below:
```bash
install optimum intel stable diffusion
...
```

```python
# import optimum-intel stable diffusion
# run optimization
...
```

Also we could make a doc page below one of those:  https://huggingface.co/docs/diffusers/optimization/mps ?

Comment (hshen14) [reply README.md]: Thanks @patrickvonplaten for your comments. Seems there is some issue in my code sync in this PR so I created a new PR (https://github.com/huggingface/diffusers/pull/1324) to make PR clean. Sample code was added. Please help review from there.

BTW, could you please share the instructions on how to create a doc page?

**Comment by hshen14** [comment]
> > Hi @echarlaix , it might be better to leave users to decide how to use quantization APIs. For simplicity, no accuracy-aware looks better; for accuracy, need accuracy driven strategy. We may provide both options to users and start from simple one.
> 
> I find the accuracy strategy pretty difficult to use currently : it increases a lot the time of the quantization process, gives no control on what modules are being quantized and will fail if no model within the given tolerance is found. Now would be a good time to talk about it while we do the refactorization (out of scope here)

Thanks Ella. Sure, we can discuss in our sync meeting.

**Pull request closed without merging**